ecl/hthor/hthor.cpp

```diff
@@ -3801,21 +3801,32 @@ void CHThorGroupSortActivity::createSorter()
         return;
     }
     if(stricmp(algoname, "quicksort") == 0)
+    {
         if((flags & TAFstable) != 0)
             sorter.setown(new CStableQuickSorter(helper.queryCompare(), queryRowManager(), InitialSortElements, CommitStep, this));
         else
             sorter.setown(new CQuickSorter(helper.queryCompare(), queryRowManager(), InitialSortElements, CommitStep));
+    }
     else if(stricmp(algoname, "parquicksort") == 0)
         sorter.setown(new CParallelStableQuickSorter(helper.queryCompare(), queryRowManager(), InitialSortElements, CommitStep, this));
     else if(stricmp(algoname, "mergesort") == 0)
-        sorter.setown(new CStableMergeSorter(helper.queryCompare(), queryRowManager(), InitialSortElements, CommitStep, this));
+    {
+        if((flags & TAFparallel) != 0)
```

Comment:
Seems a little odd for ,PARALLEL to toggle for merge, but default is parallel algorithm anyway (i.e. qsort).
Might lead readers of the ECL into believing that without ,PARALLEL it is not a parallel sort..
Perhaps should just leave it as a algorithm choice (and make it the default at some point)?